target.go

```diff
@@ -42,6 +42,12 @@ func (t *Target) validate() error {
 		}
 	}
 
+	if t.Image != nil {
```

Comment:
Let's add a `validate` to `t.Image`, which itself could `return nil` if `*Image` is `nil` (note: this is what I did in #492

Reply:
done